xclip.c

```diff
@@ -176,6 +176,14 @@ int clean_requestors() {
 static void
 doOptMain(int argc, char *argv[])
 {
+    /* Support both -option and --option form. */
+    int i;
+    for (i = 1; i < argc; ++i) {
```

Comment:
For code hygiene, I suggest declaring the counter variable within the for loop. E.g., `for (int i = 1 ...`

Reply:
I thought it should be compiled with ANSI C compiler. I'll use `for (int i =...` style if it allowed.